auth-app/build.gradle

```diff
@@ -1,101 +1,36 @@
-apply plugin: 'war'
-apply plugin: 'eclipse-wtp'
-apply plugin: 'jacoco'
-
-jacoco {
-  toolVersion = "0.8.4"
+plugins {
+	id 'org.springframework.boot' version '2.3.2.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+	id 'java'
 }
 
-sourceCompatibility = 1.8
+group = 'gameontext'
+version = '0.0.1-SNAPSHOT'
```

Comment:
we're at 0.0.7 at least.. :-P
In seriousness, we should probably version all of these apps beyond 0.0.1 ..

Reply:
Aye maybe, not this time round tho.